bcgov/PIMS
# PIMS-1822/25 Property Errors

## 🎯 Summary

  
[PIMS-1822](https://citz-imb.atlassian.net/browse/PIMS-1822)
[PIMS-1825](https://citz-imb.atlassian.net/browse/PIMS-1825)


## Changes
- Property Table only loads properties relevant to a user's agencies if not admin/auditor
- Trying to view Property Details of a property that doesn't exist or is outside of general user's agencies redirects to "/" page
- Suppress extra LTSA calls when pid is 0
- Add entry for classification lookup that handles undefined key.

I would like to find a more standard solution to viewing the request status from the data loader in the future. Maybe even the .ok property of the response as well. This should work until then.    

## Testing
- Try this with both admin and general user roles
- View property table, filter by agency -> should see different results depending on role
- View property details pages for properties in your agency and outside of it.


## 🔰 Checklist


> - I have performed a self-review of my code.
> - I have commented my code, particularly in hard-to-understand areas.
> - I have made corresponding changes to the documentation where required.
> - I have tested my changes to the best of my ability.
> - My changes generate no new warnings.
